### PR TITLE
QOLs balloon alerts to the armour booster module

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request
Adds balloon alerts for the armour booster module on toggle
image

## Why It's Good For The Game

The simple clarity is helpful and it works if you somehow get the module on a suit that doesn't normally get it (where there's no helmet icon differences)
Also makes it extra clear for newer players who probably don't understand how it works (like fluke ops)

## Changelog
:cl:
qol: Toggling an armour booster module on a MODsuit now gives a balloon alert making the tradeoffs more clear.
/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
armor*

File: code/modules/mod/modules/modules_antag.dm [hunk 1]
```diff
@@ -46,6 +46,7 @@
 	if(!.)
 		return
 	playsound(src, 'sound/mecha/mechmove03.ogg', 25, TRUE, SHORT_RANGE_SOUND_EXTRARANGE)
+	balloon_alert(mod.wearer, "armour boosted, EVA lost!")
```
Comment (Mothblocks) [inline code/modules/mod/modules/modules_antag.dm:49]:
```suggestion
	balloon_alert(mod.wearer, "armour boosted, EVA lost")
```

exclamation point is for failure

File: code/modules/mod/modules/modules_antag.dm [hunk 2]
```diff
@@ -65,6 +66,7 @@
 		return
 	if(!deleting)
 		playsound(src, 'sound/mecha/mechmove03.ogg', 25, TRUE, SHORT_RANGE_SOUND_EXTRARANGE)
+		balloon_alert(mod.wearer, "armour retracts, EVA ready!")
```
Comment (Mothblocks) [inline code/modules/mod/modules/modules_antag.dm:69]:
```suggestion
		balloon_alert(mod.wearer, "armour retracts, EVA ready")
```

exclamation point is for failure

**Review by Mothblocks — CHANGES_REQUESTED** [review]
considering the item in game is called armor, changing armour to armor would be preferred but not required

**Comment by RikuTheKiller** [comment]
It's a bit long and since balloon alerts stick around for a while, I'd rather not stick that on my screen mid-fight, but oh well.

**Comment by Mothblocks** [comment]
in combat is where they're some of the most important compared to chat, because you need to keep your eyes on the combat

**Comment by Mothblocks** [comment]
shortening would be ok though

**Comment by YakumoChen** [comment]
I don't really know how I'd shorten it since both the armour and EVA aspects are pretty important game information

**Pull request merged**